raiden/network/transport/matrix/client.py

```diff
@@ -13,11 +13,15 @@
 from matrix_client.errors import MatrixHttpLibError, MatrixRequestError
 from matrix_client.room import Room as MatrixRoom
 from matrix_client.user import User
+from requests import Response
 from requests.adapters import HTTPAdapter
 
 log = structlog.get_logger(__name__)
 
 
+SHUTDOWN_TIMEOUT = 35
```

Comment:
This seems quite long, what's the reason for this?

Reply:
The matrix long-polling requests have a 30s timeout. 
Normally that shouldn't matter since the sync greenlet is aborted with `gevent.kill()` on transport stop. But just to be on the safe side I chose a timeout that was slightly larger then those 30s.